src/icons/folderIcons.ts

```diff
@@ -329,6 +329,15 @@ export const folderIcons: FolderTheme[] = [
         ],
       },
       { name: 'folder-less', folderNames: ['less'] },
+      {
+        name: 'folder-gulp',
+        folderNames: [
+          'gulpfile.js',
```

Comment:
Just wondering if you really name your folders "gulpfile.babel.js"? What files are inside of this directory? I would've suggested that the folder name could be "gulp" and inside of this folder you have files like "gulpfile.js" or "gulpfile.babel.js".

Reply:
I honestly didn't understood what you meant, here.

Were you saying that using Node's module resolution to replace files with folders is not a good pattern and you won't support it?

I have done as you suggested, anyways.